api/GovStack_Registration_BB_API.json

```diff
@@ -267,15 +264,15 @@
 			}
 		},
 
-		"/service/applicationFileList": {
+		"/applications": {
 			"get": {
-				"tags": ["Applicant"],
-				"summary": "Retrieve all applicant's application files",
-				"description": "Returns application list",
+				"tags": ["Applicant", "Operator"],
+				"summary": "Retrieve all application files",
+				"description": "Returns a list of application files the user has permission to access (i.e. either only the applicant's own applications or all applications for an operator to process)",
 				"operationId": "getApplicantFileList",
 				"parameters": [{
-						"$ref": "#/components/parameters/Information-Mediator-Client"
-					},
+					"$ref": "#/components/parameters/Information-Mediator-Client"
+				},
 					{
```

Comment:
Are the generic query parameters for sorting + pagination (e.g. firstResult, maxResult, ...) important enough for a BB spec at the higher level? Or would it be easier to read if we remove these and only specify things related to functional aspects of the BB (+ maybe add a general para in the GitBook that these querying details should be supported)?